Dockerfile

```diff
@@ -12,10 +12,10 @@ RUN update-alternatives --set python /usr/bin/python3.6
 RUN pip3 -q install pip --upgrade
 
 # Install HB
-RUN pip3 install hummingbird-ml
+RUN pip3 install hummingbird-ml[docs,tests,extra]
 
-# Install HB dev tools
-RUN pip3 install flake8 coverage autopep8 jupyter pre-commit pytest
+# Install additional dev gools for notebook env
```

Comment:
goals?

Reply:
This dockerfile is sort of bloated and needs to be rewritten to have a separate build phase to not have all of the extra  stuff.  But...as it is is, it already has all of the extra stuff.

I think i'll open a "good first issue" for this.

Reply:
No I mean, is gools a typo?

Reply:
LOL!  right.  i'll fix.  :-D